File: JavaScript/7js 1/generate_ga.js
/*
|--------------------------------------------------------------------------
| Genetic Algorithm Series - #1 Generate
|--------------------------------------------------------------------------
|
*/

function generate(length) {
   let res = "";
   for (let i = 0; i < length; i++) res += Math.round(Math.random());
   return res;
}


function generate(length) {
   return [...Array(length)].map((val) => (Math.random() * 2) ^ 0).join(``);
}


function generate(length) {
   return Array.from(Array(length), (x) => Math.floor(Math.random() * 2)).join("");
}


console.log(generate(16), 16);
console.log(generate(32), 32);
console.log(generate(64), 64);

File: Python/7py 1/generate_ga.py
"""
|--------------------------------------------------------------------------
| Genetic Algorithm Series - #1 Generate
|--------------------------------------------------------------------------
|
"""

import random
def generate(length):
   return ''.join([str(round(random.random())) for i in range(length)])


def generate(length):
   return ''.join(random.choice('01') for i in range(length))


def generate(length):
   return ''.join([str(random.randint(0, 1)) for i in range(length)])


print(generate(16), 16)
print(generate(32), 32)
print(generate(64), 64)